### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by nerfZael**

### CODE REVIEW

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap.py,agents/token/test_swap_and_send.py 10

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap.py::test_auto_tx_swap_multiple_2 10

**Comment by nerfZael** [comment]
/workflows/benchmarks agents/token/test_swap.py::test_auto_tx_swap_multiple_2 10

File: benchmarks.json [hunk 1]
```diff
@@ -2,23 +2,29 @@
...
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_tokens_from_base": "90.00",
-        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_multiple": "100.00",
-        "autotx/tests/agents/token/test_swap_and_send.py::test_auto_tx_swap_and_send_complex": "100.00",
-        "autotx/tests/agents/token/test_swap.py::test_auto_tx_swap_with_non_default_token": "100.00",
         "autotx/tests/agents/token/research/test_research.py::test_get_top_5_most_traded_tokens_from_l1": "70.00",
-        "autotx/tests/agents/token/research/test_research.py::test_get_token_exchanges": "90.00",
-        "autotx/tests/agents/token/send/test_send.py::test_auto_tx_send_erc20_parallel": "100.00"
+        "autotx/tests/agents/token/research/test_research.py::test_get_token_exchanges": "90.00"
     },
     "iterations": 1
```
Comment (cbrzn) [inline benchmarks.json:29]: nit: should we add a benchmarks json with 10 iterations?

**Review by cbrzn — APPROVED** [review]

File: benchmarks.json (hunk 1 above)
Comment (nerfZael) [reply benchmarks.json:29]: It's not used for anything right now, so I wouldn't bother. Not sure why I implemented it

Comment (nerfZael) [reply benchmarks.json]: Updated it anyway

**Pull request merged**